resources/assets/coffee/react/profile-page/main.coffee

```diff
@@ -84,6 +84,7 @@ class ProfilePage.Main extends React.PureComponent
     $(@pages).sortable
       cursor: 'move'
       handle: '.js-profile-page-extra--sortable-handle'
+      items: '.js-sortable--page:not([data-page-id="account_standing"])'
```

Comment:
add isSortablePage function and check that in the function/loop instead (also for tabs).